progress-reports/icon-bridge/BTP-August-2022-Progress-Update.md

```diff
@@ -0,0 +1,53 @@
+| Update bsc relay config parameters | Released | Hard-coded config params to be reflected in the deployment environment | https://github.com/icon-project/icon-bridge/issues/161 |
+| Add useful logging to relay | Released | Add logging data necessary to debug issues on the relay | https://github.com/icon-project/icon-bridge/issues/159 |
+| Get receipts from blocks| Released | Reduce the number of RPC calls | https://github.com/icon-project/icon-bridge/issues/240 |
+| BSC header verification | Released | improve security and conform to the definition of having chained blocks | https://github.com/icon-project/icon-bridge/issues/243 |
+| Improve slack message formatting | Released | Improve readability of slack log messages | https://github.com/icon-project/icon-bridge/issues/214 |
+| Sanitize E2E framework code | Released | Remove mention of keys, including test units, and hard-coded values | https://github.com/icon-project/icon-bridge/issues/204 |
+| Include uint256 as maximum value for transactions | Released |  Smart contracts must conform to common integer bound | https://github.com/icon-project/icon-bridge/issues/195 |
+| Use filter query to check if BTP message exists in chain | Released |  Enhancement to reduce the number of RPC calls | https://github.com/icon-project/icon-bridge/issues/305 |
+| TransferBatch check if input request empty | Released |  Avoid wasteful transactions especially on the side of BMR | https://github.com/icon-project/icon-bridge/issues/251 |
+| Duplication of Serial Number  | Released |  Serial numbers should be unique even after contract upgrade | https://github.com/icon-project/icon-bridge/issues/324 |
+| Update block confirmation interval | Released |  Issue fix regarding block verifier failing and gets stuck in a loop causing the relay to become stuck | https://github.com/icon-project/icon-bridge/issues/322|
+
+
+
+## Deliverables Projected for Next Cycle
+
+
+| Name | Development State | Notes |
+|:-----|:------------------|:-----|
+| Documentation for the Icon Bridge Repo | Almost complete | This is almost complete and is running through internal reviews. This will be an ongoing task to ensure new users of the repo can understand the codebase more easily |
+| Improve code coverage of the code base | Not Started | The codecoverage of the repo is low. Work towards achieving the target of 80% coverage to improve the health of the overall codebase. |
+| Assist with Security Audit | In Progress | The Security Audit has started. Any information needed to be provided or fixes needed based on security audit review will be provided adhoc.|
+| Assist with Near release and integration | In Progress | Near integration deployment is being planned September Release. Team to assist. |
```

Comment:
Please specify that this is NEAR testnet.
Similarly, add a line about assisting with SNOW testnet and mainnet.